cloudslang-all/src/main/java/io/cloudslang/lang/api/Slang.java

```diff
@@ -40,7 +40,7 @@
 	 * @param systemProperties the system properties for the flow or operation run
 	 * @return the execution ID in score
 	 */
-	public Long run(CompilationArtifact compilationArtifact, Map<String, ? extends Serializable> runInputs, Map<String, ? extends Serializable> systemProperties);
+	public Long run(CompilationArtifact compilationArtifact, Map<String, ? extends Serializable> runInputs, Map<String, String> systemProperties);
```

Comment:
SP values will be strings (SP will be objects to support features like encryption)